### ISSUE TITLE
javascript error for unstaged resources

### ISSUE THREAD

**Issue by ttyiav** [BobRay/StageCoach#4]
The regClientStartupHTMLBlock call at
https://github.com/BobRay/StageCoach/blob/463e53495609f2c6480c9bb2d2a88ff2247ded9d/core/components/stagecoach/elements/plugins/stagecoach.plugin.php#L98

does not check whether the StageCoach's TVs are bound (available) to the current resource or not.

Therefore Javascript code:
```
var stageDateTv = document.getElementById("tv' . $stageDateTvId . '").value;
var stagedResourceTv = document.getElementById("tv' . $stagedResourceTvId . '").value;
```
cause js-errors in MODX manager for not staged resources, because the 'value' is requested from 'undefined'.

Quick fix is to check the existence of the DOM elements tv{$stageDateTvId} and tv{$stagedResourceTvId}.

```
var getStageDateTv = document.getElementById("tv' . $stageDateTvId . '");
var getstagedResourceTv = document.getElementById("tv' . $stagedResourceTvId . '");
var stageDateTv, stagedResourceTv;
if (getStageDateTv && getstagedResourceTv) {
    stageDateTv = getStageDateTv.value;
    stagedResourceTv = getstagedResourceTv.value;
}
```

**Comment by BobRay** [comment]
Good catch, thanks.

Do you think it would work to have the plugin check for a modTemplateVarTemplate to see if the TVs are connected to the resource's template? I'm thinking that would be more efficient.

**Comment by BobRay** [comment]
Fixed in Version 1.5.3  -- Checks modTemplateVarTemplate.
